src/covid_model_seiir_pipeline/lib/workflow/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .specification import (
+    TaskSpecification,
+    WorkflowSpecification,
+)
+from .template import (
+    TaskTemplate,
+    WorkflowTemplate
```

Comment:
Why are you mixing relative an absolute imports between submodules?

Reply:
I've only got relative imports in the package inits.  The idea is that they:
- declare the subpackage api (obviously)
- enforce the subpackage structure
- Only know local structure.  Which I was thinking would make them easy to move around as a unit without code churn.  Given that the module use absolute imports, this seems like a dumb idea though.

Reply:
I had a reason, but it's dumb on reflection.

Reply:
Fixed